client/js/components/statement/StatementExportModal.vue

```diff
@@ -54,24 +55,45 @@
         <div class="grid grid-cols-5 gap-3 mt-1 mb-5">
           <dp-input
             v-for="(column, key) in docxColumns"
-            :key="key"
             :id="key"
-            type="text"
+            :key="key"
+            v-model="column.title"
             :data-cy="column.dataCy"
             :placeholder="Translator.trans(column.placeholder)"
-            :width="column.width"
-            v-model="column.title" />
+            type="text"
+            :width="column.width" />
         </div>
+        <template v-if="this.active === 'zip' || isSingleStatementExport">
+          <h3
+            id="docxFileName"
+            class="inline-block text-lg mr-1">
```

Comment:
`text-lg` is not yet configured to use demosplan sizing. This will be soon rolled out, changing the value of `text-lg` to 19.2px. However in the designs i meant those to be "label-like" in terms of font-size and font-weight (16px / segoeUi semibold). When seeing this on a Debian System i see that this is where system fonts make it hard to match the design, as Ubuntu uses `DejaVu Sans` which has no weight of 600 which is semibold, leading to the bold weight to be used. I feel this needs to be tackled globally, i guess by ditching system fonts entirely. For the time being i think `text-lg` can be used.